extensions/amp-story-360/0.1/amp-story-360.js

```diff
@@ -380,6 +381,16 @@ export class AmpStory360 extends AMP.BaseElement {
     );
   }
 
+  /** @private */
+  onPageNavigation_() {
+    if (this.isOnActivePage_) {
+      this.play_();
```

Comment:
Do you need to keep the `canAnimate` test? The `userAssert` will throw an error

Reply:
Good point. I added the check in `play_` so it's the same pattern as in `rewind_`